### ISSUE TITLE
Install Profile sites have two Alternative Title fields.

### ISSUE THREAD

**Issue by rosiel** [Born-Digital-US/islandora_install_profile_demo#26]
On the Sandbox, as of today, Repository Item has both field_alt_title and field_alternative_title. They're both named Alternative Title. Only one shows up on the form and the other is Disabled, but I can't tell which.
<img width="992" alt="Manage Fields page from Sandbox showing multiple 'Alternative Title' fields" src="https://user-images.githubusercontent.com/1943338/186966377-9fafa503-bb4f-4c0e-845b-971cfd4fc284.png">

**Comment by noahwsmith** [comment]
My first question would be whether this happens with a new install, or if
this is the result of an experiment someone did on Sandbox?

--

Noah Smith

Founder + CEO

Pronouns: he/him/his. Scheduling a meeting with me? Set something up with
Calendly <https://calendly.com/noah-born-digital>
Born-Digital | 84 Russell St, Hadley MA | (413) 259-6777 | born-digital.com

**Comment by rosiel** [comment]
No. [We changed the name of the field last July](https://github.com/Islandora/islandora_defaults/commit/b1179983eaf43cc4013aabc703763bea0423975a#diff-522098abc5618c9d74bbbf40067f109de49f981a321dfd3b217f45b87e77a06eL5) (over a year ago). If you install Defaults and then the Install Profile's config (which wasn't updated to know about field_alt_title, AFAIK) then you'd get this scenario. I don't think this is someone messing around with sandbox.

**Comment by noahwsmith** [comment]
Looks like you're right. It appears that this is a global find-and-replace job for these hits
https://github.com/Islandora-Devops/islandora_install_profile_demo/search?p=1&q=field_alternative_title
